From a7996761d1dacbac10c892c16faa90dfe46affcd Mon Sep 17 00:00:00 2001
From: Steve Lin <steven.lin1@broadcom.com>
Date: Wed, 15 Mar 2017 16:59:35 -0400
Subject: [PATCH] ARM: dts: BCM953012HR: Add ethernet aliases

Adding ethernet aliases.  These are used, for example, by bootloaders,
to modify the MAC addresses in the device tree.

Signed-off-by: Steve Lin <steven.lin1@broadcom.com>
Acked-by: Jon Mason <jon.mason@broadcom.com>
Signed-off-by: Florian Fainelli <f.fainelli@gmail.com>
---
 arch/arm/boot/dts/bcm953012hr.dts | 3 +++
 1 file changed, 3 insertions(+)

--- a/arch/arm/boot/dts/bcm953012hr.dts
+++ b/arch/arm/boot/dts/bcm953012hr.dts
@@ -41,6 +41,9 @@
 
 	aliases {
 		serial0 = &uart0;
+		ethernet0 = &gmac0;
+		ethernet1 = &gmac1;
+		ethernet2 = &gmac2;
 	};
 
 	chosen {
